templates/zerver/help/browse-and-join-streams.md

```diff
@@ -29,7 +29,8 @@ following image.
 
 ![Streams subscribe alert](/static/images/help/stream-subscribe.png)
 
-The message will present to you the option of subscribing to the stream, and you can do so by simply clicking the gray button.
```

Comment:
Line 13 renders like this:
![Uploading Screen Shot 2017-01-06 at 10.30.09 am.png…]()

I guess you need to fix the indentation of 'Here'.